libweb3jsonrpc/Debug.cpp

```diff
@@ -134,3 +137,34 @@ Json::Value Debug::debug_storageAt(string const& _blockHashOrNumber, int _txInde
 		ret[toHex(toCompactBigEndian(i.first, 1), 2, HexPrefix::Add)] = toHex(toCompactBigEndian(i.second, 1), 2, HexPrefix::Add);
 	return ret;
 }
+
+Json::Value Debug::debug_traceCall(Json::Value const& _call, std::string const& _blockNumber, Json::Value const& _options)
+{
+	Json::Value ret;
+	try
+	{
+		Block temp = m_eth.asOf(jsToBlockNumber(_blockNumber));
+		TransactionSkeleton ts = toTransactionSkeleton(_call);
+		if (!ts.from) {
+			ts.from = m_ethAccounts.defaultTransactAccount();
+		}
+		u256 nonce = temp.transactionsFrom(ts.from);
+		u256 gas = ts.gas == Invalid256 ? m_eth.gasLimitRemaining() : ts.gas;
+		u256 gasPrice = ts.gasPrice == Invalid256 ? m_eth.gasBidPrice() : ts.gasPrice;
+		Transaction t(ts.value, gasPrice, gas, ts.to, ts.data, nonce);
```

Comment:
At some other point, I have seen code that adds funding to the sender if it does not have enough. Is that the case for `eth_call`? If yes, we should also add that here.